package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cision/rover-ui",
-  "version": "2.0.0-beta.5",
+  "version": "2.0.0-beta.6",
```

Comment:
We should establish a the pattern for when to bump versions. I'm fine with this here, but I think a commit of it's own is warranted (especially to keep the CHANGELOG consistent and properly up-to-date)

Reply:
Oh, snap, I forgot about the changelog.

I don't mind reverting that mit and making a separate PR for the version bump. Is there a special incantation I need to use to make the changelog nice?

Reply:
It's just `yarn update-changelog`. My current workflow for releasing new versions has been.

* `git checkout master`
* Bump version in package.json
* `yarn update-changelog`
* `git commit -m "v{version}"`
* `git push`
* `yarn release --tag next`